Makefile

```
@@ -327,7 +327,15 @@ release-%:
 test: generate fmt vet manifests build-cli-mocks ## Run tests
 	## Skip running TKG integration tests
 	$(MAKE) ytt -C $(TOOLS_DIR)
+
+	## Test the YTT cluster templates
+	echo "Changing into the provider test directory to verify ytt cluster templates ..."
+	cd ./pkg/v1/providers/tests/unit && PATH=$(abspath hack/tools/bin):$(PATH) $(GO) test -v -timeout 30s ./
```

```suggestion
	cd ./pkg/v1/providers/tests/unit && PATH=$(abspath hack/tools/bin):"$(PATH)" $(GO) test -v -timeout 30s ./
```
To handle paths with directories with spaces in them